tests/cpp_api/multiple_dataloaders_test/README.md

```diff
@@ -0,0 +1,23 @@
+# multiple_dataloaders_test application
+This application demonstrates a basic usage of rocAL's C API to read numpy files from disk and decode them using multiple dataloaders and apply different augmentation in each pipeline.
+
+## Build Instructions
+
+### Pre-requisites
+*  Ubuntu 20.04/22.04 Linux
```

Comment:
ROCm supports ub22 & ub24. There is no more ub20. Please update

Reply:
Done